hw_7.py

```diff
@@ -0,0 +1,14 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('file/file.txt', 'r') as f:
+        count = 0
+        for x in f.readlines():
```

Comment:
It's too complicated. There is an easier solution.

Reply:
Also, we don't need this additional iteration through all these lines.

Reply:
fixed